queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019; 
+SELECt name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE (weight_kg BETWEEN 10.4 AND 17.3) OR weight_kg = 10.4 OR weight_kg = 17.3;
```

Comment:
Great job so far 👍 . According to the requirement, this query needs to find all animals whose weights fall between 10.4 and 17.3, which means 10.4 and 17.3 are inclusive. Kindly update the query and submit it back. 
I think you can just remove the OR part of the query above and you will get your answer 😉 .

Reply:
Thank you @Lizdev-05 for the remark, I'll correct it right away